lib/cadet_web/views/assessments_view.ex

```diff
@@ -73,7 +73,7 @@ defmodule CadetWeb.AssessmentsView do
        }) do
     components = [
       build_question(%{question: question}),
-      build_answer_by_question_type(%{question: question}),
+      build_answer_and_comment_by_question_type(%{question: question}),
```

Comment:
What do you think about renaming this to just `build_answer_fields_by_question_type`? I foresee us adding more to this in future.

Reply:
Makes sense. Done